ui/onboarding/Keycard/CreatePINModal.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Dialogs 1.3
+import StatusQ.Controls 0.1
+import "../../imports"
+import "../../shared"
+import "../../shared/keycard"
+
+ModalPopup {
```

Comment:
This should use `StatusModal` instead. Can be imported from `StatusQ.Popups 0.1`